### ISSUE TITLE
Default config don't creates on Ubuntu 16.04

### ISSUE THREAD

**Issue by popstas** [popstas/transmission-cli#36]
```
$ transmission-cli tl
[critical] Config file not found: /home/popstas/.transmission-cli.yml
```

But creates on macOS.
Related: #35

**Comment by mabushey** [comment]
I noticed this too on Ubuntu. I copied the transmission-cli.dist.yml over and edited it.

**Comment by popstas** [comment]
Was closed in #35, but unreleased. Now it fixed in v0.7.0 release.
